packages/abi-typegen/src/utils/getPackageVersion.test.ts

```diff
@@ -0,0 +1,10 @@
+import packageJson from '../../package.json';
+
+import { getPackageVersion } from './getPackageVersion';
+
+describe('getPackageVersion.ts', () => {
+  test('should get package version just fine', () => {
+    const expectedVersion = packageJson.version;
+    expect(getPackageVersion().version).toEqual(expectedVersion);
```

Comment:
Here I am looking for the `package.json` inside the `abi-typegen` package, not the root one.

If we use the `fuels` one, it will fail when installing the package in a standalone fashion. Right?

Reply:
> **Note**: I changed the root `package.json:version` to `0.0.0`